apps/prairielearn/src/lib/editors.ts

```diff
@@ -1988,7 +1997,10 @@ export class CourseInfoCreateEditor extends Editor {
     // This will error if:
     // - this.course.path does not exist (use of writeJson)
     // - Creating a new file and infoPath does exist (use of 'wx')
```

Comment:
Nit, move this comment above the `writeFile` line and update the reference to `writeJson`.